persistence/test/application_test.go

```diff
@@ -12,15 +12,15 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
-func FuzzApplication(f *testing.F) {
+func fuzzApplication(f *testing.F) {
 	fuzzSingleProtocolActor(f,
-		NewTestGenericActor(types.ApplicationActor, newTestApp),
-		GetGenericActor(types.ApplicationActor, getTestApp),
+		newTestGenericActor(types.ApplicationActor, newTestApp),
+		getGenericActor(types.ApplicationActor, getTestApp),
 		types.ApplicationActor)
 }
 
-func TestInsertAppAndExists(t *testing.T) {
-	db := NewTestPostgresContext(t, 0)
+func testInsertAppAndExists(t *testing.T) {
```

Comment:
Tests should be capitalized. Leaving one comment but dItto everywhere else: https://go.dev/doc/tutorial/add-a-test